middlewares/verifyWebhook.js

```diff
@@ -2,8 +2,8 @@ const portal = require("@zegal/apiportal-utils");
 async function verifyWebhook(req, res, next) {
   try {
     await portal.verifyWebhookEvent(
-      req.body,
-      req.headers["X-Dragon-Law-Signature"],
+      JSON.stringify(req.body),
```

Comment:
what is this for ?

Reply:
we need to send the stringified payload so that the string can be added with timestamp to generate hash